### PR TITLE
LOG-3570: Remove tlsSecurityProfile feature gate

### PR DESCRIPTION
**Pull request by syedriko**
### Description

Remove tlsSecurityProfile feature gate

/cc @cahartma 
/assign @jcantrill 

### Links
- JIRA: https://issues.redhat.com/browse/LOG-3570

### CODE REVIEW

File: internal/generator/vector/outputs.go [hunk 1]
```diff
@@ -68,15 +67,13 @@ func Outputs(clspec *logging.CollectionSpec, secrets map[string]*corev1.Secret,
...
-				op[generator.Ciphers] = ""
-			} else {
-				outMinTlsVersion, outCiphers := op.TLSProfileInfo(o, ",")
-				op[generator.MinTLSVersion] = outMinTlsVersion
-				op[generator.Ciphers] = outCiphers
-			}
+		if o.Name == logging.OutputNameDefault && o.Type == logging.OutputTypeElasticsearch {
+			op[generator.MinTLSVersion] = ""
+			op[generator.Ciphers] = ""
+		} else {
```
Comment (cahartma) [inline internal/generator/vector/outputs.go:73]: Is this going to set tls for all other outputs?   Don't you still need to check for loki here?

Comment (syedriko) [reply internal/generator/vector/outputs.go:73]: This is going to set TLS version and ciphersuites for the all the outputs but the default ES, if they enable TLS.
The line I removed said `if the feature gate flag is set or it's loki`, [docs/features/tls_security_profile.adoc] ^^^ had this `The feature gate is ignored for the default log storage of type loki.` Without feature gate, there's no need for loki exception.

**Review by jcantrill — COMMENTED** [review]
/lgtm

**Pull request merged**